tests/dutyTestData.ts

```diff
@@ -0,0 +1,49 @@
+import { ObjectId } from 'mongodb';
+
+const DutyToBeAddedForPOSTtest = {
+  name: 'testDuty1',
+  description: 'o(*￣▽￣*)ブ',
```

Comment:
Please this isn't funny its just a waste of time... You could for instance, spend your time writing quality code instead of making sure there is real gibberish in your otherwise unreadable code...

If you want to add jokes to your code, make sure it is good, clean code... I don't laugh when I see this, but rather sigh in exasparation at the fact that this is what you saw fit to spend your time on...

please PLEASE just make your code better instead of joking around